src/librustc_yk_sections/mir_cfg.rs

```diff
@@ -7,64 +7,46 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-/// Custom CFG serialiser for Yorick.
-/// At the time of writing no crate using `proc_macro` can be used in-compiler, otherwise we'd have
-/// used Serde.
+#![allow(unused_imports)]
+
+//! MIR to TIR converter/serialiser.
+//!
+//! We convert Rust's MIR into our own IR we call "TIR" (tracing IR), which is then stashed away in
```

Comment:
"Our own" feels odd. How about something like "This module converts rustc MIR into Yorick TIR (Tracing IR). TIR is more suitable for the run-time tracer: TIR is in SSA form (but it does preserve MIR's block structure)."?